eclide/ChildBuilderFrame.cpp

```diff
@@ -404,7 +408,27 @@ class CBuilderFrame :
             IAttributeVector attrs;
             Dali::CEclExceptionVector errors;
             IAttributeBookkeep attrProcessed;
-            attr->PreProcess(PREPROCESS_SUBMIT, _ecl, attrs, attrProcessed, errors);
+            PREPROCESS_TYPE processType = PREPROCESS_UNKNOWN;
+            switch (action) {
+            case Dali::WUActionGenerate:
+                if (!isSaved)
+                    m_dlgview.DoGenerate();
+                processType = PREPROCESS_GENERATE;
+                break;
+            case Dali::WUActionCustom1:
```

Comment:
This change looks reasonable. Not sure why these actions are qualified with "Dali", they don't seem dali related, but that's minor.